deployment-scripts/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ x-service-variables: &common-creds
     DEEPFENCE_POSTGRES_USER_DB_PORT: 5432
     DEEPFENCE_POSTGRES_USER_DB_SSLMODE: disable
     DEEPFENCE_KAFKA_BROKERS: deepfence-kafka-broker:9092
-    DEEPFENCE_MINIO_BUCKET: deepfence
+    DEEPFENCE_MINIO_BUCKET: default
```

Comment:
What is the problem using deepfence instead?
Will that break existing deployed solutions?

Reply:
yes, setting it to `deepfence` will break existing deployments
the existing bucket is `default` if we set it to `deepfence` the bucket changes, which will make existing data inaccessible